packages/sdk/src/grpc/translation.ts

```diff
@@ -2469,6 +2469,8 @@ function trCatchupStatus(catchupStatus: GRPC.PeersInfo_Peer_CatchupStatus): SDK.
             return SDK.NodeCatchupStatus.Pending;
         case CatchupStatus.UPTODATE:
             return SDK.NodeCatchupStatus.UpToDate;
+        default:
+            return null;
```

Comment:
A bit late to the party here, but instead we should use 
```suggestion
        case undefined:
            return null;
```

This will help us notice new variants when updating the SDK with a new reference to the protobuf definitions